apollo-runtime/api.txt

```diff
@@ -373,38 +373,65 @@ package com.apollographql.apollo.response {
 
 package com.apollographql.apollo.subscription {
 
+  public final class JsonWritersKt {
+  }
+
   public interface OnSubscriptionManagerStateChangeListener {
     method public void onStateChange(com.apollographql.apollo.subscription.SubscriptionManagerState!, com.apollographql.apollo.subscription.SubscriptionManagerState!);
   }
 
-  public abstract class OperationClientMessage {
-    method public String toJsonString();
-    method public abstract void writeToJson(com.apollographql.apollo.api.internal.json.JsonWriter);
+  public abstract sealed class OperationClientMessage {
```

Comment:
All of the changes here are fine since it was not possible to subclass this class anyway.